### PR TITLE
WIP: Clone Analytics

### PR DESCRIPTION
**Pull request by jiffyclub**
Adds the analytics repo to airflow deployments so that we can run the ATL.

cc @zharben @jasminetsai @gwax

### CODE REVIEW

File: airflow/airflow.cfg [hunk 1]
```diff
@@ -24,7 +24,7 @@ sql_alchemy_conn = ${AIRFLOW_META_DBURL}
 # The amount of parallelism as a setting to the executor. This defines
 # the max number of task instances that should run simultaneously
 # on this airflow installation
-parallelism = 32
+parallelism = 2
```
Comment (vansivallab) [inline airflow/airflow.cfg]: why?

File: airflow/airflow.cfg [hunk 2]
```diff
@@ -76,7 +77,7 @@ celery_app_name = airflow.executors.celery_executor
 # "airflow worker" command. This defines the number of task instances that
 # a worker will take, so size up your workers based on the resources on
 # your worker box and the nature of your tasks
-celeryd_concurrency = 16
+celeryd_concurrency = 1
```
Comment (vansivallab) [inline airflow/airflow.cfg]: why?

File: local_airflow [hunk 3]
```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env bash
```
Comment (vansivallab) [inline local_airflow]: this file probably shouldnt go into production

File: airflow/airflow.cfg (hunk 2 above)
Comment (jasminetsai) [inline airflow/airflow.cfg]: Concurrent writes slows down the ATL bc of its expensive operations

On Wednesday, February 3, 2016, Vansi V <notifications@github.com> wrote:

> In airflow/airflow.cfg
> <https://github.com/CloverHealth/aptible-airflow/pull/5#discussion_r51818756>
> :
>
> > @@ -76,7 +77,7 @@ celery_app_name = airflow.executors.celery_executor
> >  # "airflow worker" command. This defines the number of task instances that
> >  # a worker will take, so size up your workers based on the resources on
> >  # your worker box and the nature of your tasks
> > -celeryd_concurrency = 16
> > +celeryd_concurrency = 1
>
> why?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CloverHealth/aptible-airflow/pull/5/files#r51818756>.
>


-- 
Jasmine Y. Tsai
+1 415 812 2219

Comment (vansivallab) [inline airflow/airflow.cfg]: Ok there are a few ways to handle this. Changing celeryd_concurrency will affect all dags in airflow not just the atl dag. Airflow provides a few ways to handle this: concurrency parameter on a Dag and [pools](https://pythonhosted.org/airflow/concepts.html#pools)

Comment (jasminetsai) [inline airflow/airflow.cfg]: :+1:  , was this left in globally because you all had your own env set up?

Comment (jiffyclub) [inline airflow/airflow.cfg]: We've got a todo to change this back before we merge, we've been playing around with different ways of getting the optimal number of ATL tasks running concurrently.

File: airflow/airflow.cfg (hunk 1 above)
Comment (jiffyclub) [inline airflow/airflow.cfg]: We've got a todo to change this back before we merge, we've been playing around with different ways of getting the optimal number of ATL tasks running concurrently.

File: airflow/airflow.cfg (hunk 2 above)
Comment (jiffyclub) [inline airflow/airflow.cfg]: Ultimately we will be setting DAG-level concurrency and making an ATL-specific pool.

Comment (vansivallab) [inline airflow/airflow.cfg]: :+1:

**Comment by gwax** [comment]
Might it make sense to create a separate branch of this repository named "local" where local specific deployment gets configured?

File: local_airflow (hunk 3 above)
Comment (zharben) [inline local_airflow]: I've deleted this from the repo.

**Pull request merged**